OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -162,6 +164,15 @@ public static void initialize(@NonNull Application application, @NonNull Optimov
                 Optimove.fetchConfigsAndFinishOptimoveInit(application, config);
             }
         }
+
+        if (config.isPreferenceCenterConfigured()){
+            if (!config.isOptimoveConfigured()){
+                Log.e("Optimove", "Preference center requires optimove credentials");
```

Comment:
this is for UserInfo to be created and updated when customer id changes. UserInfo for sure could be refactored into "core" / not depend on even optimove credentials